File: app/src/main/java/com/zwstudio/logicpuzzlesandroid/puzzles/lighthouses/domain/LighthousesHintObject.java
package com.zwstudio.logicpuzzlesandroid.puzzles.lighthouses.domain;

import com.zwstudio.logicpuzzlesandroid.home.domain.HintState;

public class LighthousesHintObject extends LighthousesObject {
    public HintState state = HintState.Normal;
    public String objAsString() {
        return "hint";
    }
}

File: app/src/main/java/com/zwstudio/logicpuzzlesandroid/puzzles/lighthouses/domain/LighthousesHintObject.kt
package com.zwstudio.logicpuzzlesandroid.puzzles.lighthouses.domain

import com.zwstudio.logicpuzzlesandroid.home.domain.HintState

class LighthousesHintObject : LighthousesObject() {
    var state = HintState.Normal
    override fun objAsString(): String {
        return "hint"
    }
}